mmv1/products/redis/api.yaml

```diff
@@ -229,3 +229,58 @@ objects:
               output: true
               description: |
                 Sha1 Fingerprint of the certificate.
+      - !ruby/object:Api::Type::Integer
+        name: replicaCount
+        description: |
+          Optional. The number of replica nodes. The valid range for the Standard Tier with 
+          read replicas enabled is [1-5] and defaults to 2. If read replicas are not enabled
+          for a Standard Tier instance, the only valid value is 1 and the default is 1. 
+          The valid value for basic tier is 0 and the default is also 0.
+        min_version: beta
+      - !ruby/object:Api::Type::Array
+        name: nodes
+        description: |
+          Output only. Info per node.
+        output: true
+        min_version: beta
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'id'
+              output: true
+              description: |
+                Node identifying string. e.g. 'node-0', 'node-1'
+            - !ruby/object:Api::Type::String
+              name: 'zone'
+              output: true
+              description: |
+                Location of the node.
+      - !ruby/object:Api::Type::String
+        name: readEndpoint
+        description: |
+          Output only. Hostname or IP address of the exposed readonly Redis endpoint. Standard tier only.
+          Targets all healthy replica nodes in instance. Replication is asynchronous and replica nodes
+          will exhibit some lag behind the primary. Write requests must target 'host'.
+        output: true
+        min_version: beta
+      - !ruby/object:Api::Type::Integer
+        name: readEndpointPort
+        description: |
+          Output only. The port number of the exposed readonly redis endpoint. Standard tier only. 
+          Write requests should target 'port'.
+        output: true
+        min_version: beta
+      - !ruby/object:Api::Type::Enum
+        name: readReplicasMode
+        description: |
+          Optional. Read replica mode. Can only be specified when trying to create the instance.
```

Comment:
readReplicasMode should also have `input: true` set here if it can only be set on create.

Reply:
done